projects/nlp/model_api_tests/hello_classification_model.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
```

Comment:
Why do you need this file in model_api_tests directory? I thought we were going to use simple_labels and read the data from a file?